### PR TITLE
Fix POST input stream

### LINKED ISSUE #112: Student API POST doesn't insert into database.

### PR DESCRIPTION
**Pull request by richardzeng1**
fixes #112

### CODE REVIEW

File: student_survey.php [hunk 1]
```diff
@@ -179,6 +179,7 @@ function select_questions($choices, $list_of_quesitons)
  */
 function handle_post($body)
 {
+    $body = json_decode(json_encode($body), true);
```
Comment (siefkenj) [inline student_survey.php]: Why do you need to encode and then decode?

Comment (richardzeng1) [reply student_survey.php]: $body is a stdClass. According to https://stackoverflow.com/questions/18576762/php-stdclass-to-array I need to encode then decode $body to convert stdClass to an array.

Comment (siefkenj) [reply student_survey.php]: Hunt down where that `stdClass` is coming from. We shouldn't be using that in our code.

Comment (richardzeng1) [reply student_survey.php]: it comes from line 14. The input stream returns a string which when I decode, gets converted to a stdClass. There's not a good way to easily parse the body.

Comment (siefkenj) [reply student_survey.php]: Add a `, true` to the `json_decode`

Comment (richardzeng1) [reply student_survey.php]: Thank you

**Pull request merged**